### PR TITLE
Whatever I found while reviewing translations (smoe:Docs29 mac 101)

### PR DESCRIPTION
**Pull request by smoe**
Mostly semantic line breaks.

### CODE REVIEW

**Comment by SebKuzminsky** [comment]
A bunch of these changes look like markup fixes and general improvements to the docs, so they should go in 2.9 I think.

**Comment by smoe** [comment]
@SebKuzminsky , thank you for spotting that. Redirected to 2.9.

File: docs/src/ladder/classic-ladder.adoc [hunk 1]
```diff
@@ -812,141 +666,107 @@ image::images/Config-io.png["Config I/O",align="center"]
...
 
 == MODBUS Settings
 
 Serial:
 
 * ClassicLadder uses RTU protocol (not ASCII).
 * 8 data bits, No parity is used, and 1 stop bit is also known as 8-N-1.
-* Baud rate must be the same for slave and master. ClassicLadder can
-  only have one baud rate so all the slaves must be set to the same rate.
+* BAUD rate must be the same for slave and master.
```
Comment (hansu) [inline docs/src/ladder/classic-ladder.adoc:724]: Baud

**Review by hansu — CHANGES_REQUESTED** [review]
:wink:

File: docs/src/ladder/classic-ladder.adoc (hunk 1 above)
Comment (smoe) [reply docs/src/ladder/classic-ladder.adoc:724]: And I was so darn confident. Want to blame some computer magazines from the 80s and thanks go to https://de.wikipedia.org/wiki/Baud .

Comment (smoe) [reply docs/src/ladder/classic-ladder.adoc]: Fixed - also reviewed other .adoc files and s/BAUD/baud/g.

**Review by hansu — APPROVED** [review]

File: docs/src/ladder/classic-ladder.adoc (hunk 1 above)
Comment (hansu) [reply docs/src/ladder/classic-ladder.adoc]: Thanks

**Pull request merged**